sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -24,82 +73,31 @@ class UniversalSpinTable(Base):
     repointing_number = Column(Integer, nullable=False)
 
 
-class FileCatalogTable:
-    """Common file catalog table"""
+class StatusTrackingTable(Base):
+    """Status tracking table"""
+
+    __tablename__ = "status_tracking_table"
+
+    id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
+    file_to_create_path = Column(String, nullable=False)
+    status = Column(statuses, nullable=False)
+    job_definition = Column(String, nullable=False)
+    ingestion_date = Column(DateTime(timezone=True), nullable=True)
+
+
+class FileCatalogTable(Base):
+    """File catalog table"""
+
+    __tablename__ = "file_catalog_table"
 
     # TODO: determine cap for strings
     id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
     file_path = Column(String, nullable=False)
-    instrument = Column(String(6), nullable=False)
-    data_level = Column(String(3), nullable=False)
+    instrument = Column(instruments, nullable=False)
+    data_level = Column(data_levels, nullable=False)
     descriptor = Column(String, nullable=False)
     start_date = Column(DateTime(timezone=True), nullable=False)
```

Comment:
Do we want timezones?

Reply:
I'd say we want to avoid timezones as much as possible and only use UTC. Does that still require a timezone=True here to clarify that this is a UTC start_date?

Reply:
That's a good question. I think I made `timezone=True` because when I previously added ingestion time to file catalog (which just got moved to file status table) I specified UTC when getting the current time and it sounds like I need to set timezone to True for it to recognize that python date time with timezone input, but I might be reading the documentation wrong. I added timezone for all the dates in case we need to specify UTC for everything. I'm not sure if I'm interpreting how that works correctly though, so I'm not sure if we really need this.

Reply:
I'm worried timezone=True will cause more problems than always assuming the timezone is UTC and having timezone=False. I think you can specify timezone when converting from this datetime to python datetimes?

Reply:
Yeah, that's true. I'll remove timezone

Reply:
Sounds reasonable, I think the only issue we'd run into is with `ingest_time`, where we'll need to make sure that is handled the same in all of our Lambda's that write to the database with `datetime.now(datetime.UTC)`.